docs/reference/organizations-on-nuget-org.md

```diff
@@ -0,0 +1,105 @@
+
+## User accounts vs. organization accounts
+
+Your user account is your identity on nuget.org and can be a member of any number of organizations. A package can belong to an organization account like it can belong to a user account. Package consumers don't see any difference between an user account or the organization account: both appear as package `owners`.
+
+An organization account has one or more user accounts as its members. These members can manage a set of packages while maintaining a single identity for ownership.
+
+## Adding a new organization
+
+To add a new organization, select your account on nuget.org, then select the **Manage Organizations...** menu command:
+
+![Menu option on nuget.org for Manager Organizations](media/org-manage-option.png)
+
+On the next page, select the **Add new organization** button:
+
+![Button to create a new organization on nuget.org](media/org-add-new-option.png)
+
+On the next page, provide the organization name and email address. Because nuget.org manages organization and user names together, the organization name must be different from any other existing organization or user accounts. The email address must also be unique across all accounts.
+
+![Add new organization page on nuget.org](media/org-add-new-page.png)
+
+Once the organization account is created, you are the administrator and can submit packages for the organization and add organization members.
+
+### Transform existing account to an organization
+
+If you're managing packages as a team using a single user account and would like to convert that account into an organization, use the **Transform your account to an organization** option on the **Manage Organizations** page:
+
+![Option on nuget.org to transform an existing account to an organization](media/org-transform-option.png)
+
+On the next page, specify different user account to assign as the administrator of the organization, then select **Transform**.
+
+![Entering information for transforming a user account to an organization](media/org-transform-page.png)
+
+> [!Warning]
+> Account conversion is irreversible: you cannot transform an organization back to a user account.
+
+## Managing organization members
+
+As the organization administrator, you can add members by providing each member's nuget.org *user account name*; email addresses cannot be used. You then mark each member as a collaborator or administrator with the following permissions:
+
+| Permission | Collaborator | Administrator |
+| --- | --- | --- |
+| Manage the organization's packages<br/>(submit new packages, update or unlist existing packages) | Yes | Yes |
+| Change organization metadata<br/>(email address, notification settings) | No | Yes |
+| Manage organization members | No | Yes |
+| Request or act on co-ownership requests for organization packages | No | Yes |
```

Comment:
I would split this into multiple sections

e.g.
1 - `Manage ownership of the organization's packages`
2 - `Handle co-ownership requests for the organization to own packages`

Reply:
Its already that way.